daemon/cluster/executor/container/container.go

```diff
@@ -225,6 +225,7 @@ func (c *containerConfig) labels() map[string]string {
 			"task":         "", // mark as cluster task
 			"task.id":      c.task.ID,
 			"task.name":    c.name(),
+			"task.slot":    strconv.FormatUint(c.task.Slot, 10),
```

Comment:
I think we should only add this if the slot number is nonzero.

Currently only replicated service tasks have slot numbers, though there is a proposal to change that: https://github.com/docker/swarmkit/issues/1554